CDTDatastore/touchdb/TDReplicator.m

```diff
@@ -147,8 +148,8 @@ - (bool)hasSameSettingsAs:(TDReplicator*)other
 {
     return _db == other->_db && $equal(_remote, other->_remote) && self.isPush == other.isPush &&
            _continuous == other->_continuous && $equal(_filterName, other->_filterName) &&
-           $equal(_filterParameters, other->_filterParameters) &&
-           $equal(_options, other->_options) && $equal(_docIDs, other->_docIDs) &&
+           $equal(_filterParameters, other->_filterParameters) && _reset == other->_reset &&
+           $equal(_heartbeat, other->_heartbeat) && $equal(_docIDs, other->_docIDs) &&
```

Comment:
We don't need to use `$equal` for `heartbeat` we can use `isEqualToNumber:` instead.